api/apps/geoprocessing/src/migrations/geoprocessing/1692254627921-AddsCostSurfacePuDataGeoDb.ts

```diff
@@ -0,0 +1,24 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddsCostSurfacePuDataGeoDb1692254627921
+  implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    /**
+     * @todo: Check how cascade deletes should work
+     */
+    await queryRunner.query(`
+      CREATE TABLE cost_surface_pu_data (
+        id uuid PRIMARY KEY DEFAULT gen_random_uuid(),
+        cost_surface_id uuid NOT NULL,
+        puid uuid UNIQUE NOT NULL references projects_pu(id),
```

Comment:
i think we can throw in an `on delete cascade`. By the time we remove `projects_pu` rows we would have already deleted the parent project, so we don't risk orphaning these `cost_surface_pu_data` rows

Reply:
Nice clarification, thanks!

Reply:
addressed f460be51ecfc5c4457dab15240ba491640dc2e28